src/components/interaction-designer/block-editors/ContactPropertySelector.vue

```diff
@@ -1,16 +1,19 @@
 <template>
-  <div class="block-contact-property form-group">
-    <label>{{'flow-builder.contact-property-label' | trans}}</label>
-    <vue-multiselect v-model="selectedProperty"
-                     track-by="id"
-                     label="displayLabel"
-                     :placeholder="'flow-builder.contact-property-selector-placeholder' | trans"
-                     :options="subscriberPropertyFields || []"
-                     :allow-empty="false"
-                     :show-labels="false"
-                     :searchable="true">
-    </vue-multiselect>
-  </div>
+  <validation-message :message-key="`block/${block.uuid}/.config.set_contact_property.property_key`" #input-control="{ isValid: isInvalid }">
```

Comment:
I think this will be `{ isValid: isInvalid }` after you update from the fix i made in vmo-3704. ;)

Reply:
Done!